packages/connectivity/src/scp-cf/connectivity-service.spec.ts

```diff
@@ -140,4 +145,28 @@ describe('connectivity-service', () => {
       done();
     });
   });
+
+  it('[MAIL] adds a proxy configuration containing at least the host, the port, and the "proxy-authorization"', async () => {
```

Comment:
[pp] I would use a `descirbe` to separate the `mail` related thest. I do not like the `[MAIL]` tag so much